src/agent/dtls_mbedtls.cpp

```diff
@@ -77,8 +77,31 @@ static void MbedtlsDebug(void *aContext, int aLevel, const char *aFile, int aLin
         aLevel = OTBR_LOG_DEBUG;
         break;
     }
+    char buf[100];
+    /* mbed inserts a EOL
+     * And so does the otbrLog()
```

Comment:
Thanks for pointing out this. Is it possible to remove them inside `otbrLog()` just like how `syslog()` does? I think this could avoid duplicated code in the commissioner side.